README.md

```diff
@@ -69,24 +81,43 @@ docker-compose -f dev-docker-compose.yml up --build
 
 This starts all necessary services using Docker Compose.
 
-### 5. Local Database Migrations & Setup
+### 6. Django setup
+
+Install the project into a virtual environment::
 
-Inside the backend container (`nsc-recommendationsp-django-1`) (or using `docker exec -it nsc-recommendationsp-django-1 bash` directlyl from your terminal):
+    python3.12 -m venv ./venv
+    source ./venv/bin/activate
+    pip3 install -r requirements-dev.txt
+
+### 7. Local Database Migrations & Setup
+
+Inside the backend container (`nsc-recommendationsp-django-1`) (or using `docker exec -it nsc-recommendationsp-django-1 bash` directly from your terminal):
 
 ```bash
 ./manage.py makemigrations
 ./manage.py migrate
 ```
+### 8. Migration for data 
```

Comment:
This needs moving, it's a useful guide but... it's not required as part of the local dev set-up. Maybe it's a doc in the /docs folder?